noticed.gemspec

```diff
@@ -19,8 +19,12 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rails", ">= 6.0.0"
   spec.add_dependency "http", ">= 4.0.0"
 
+  spec.add_development_dependency "appraisal"
   spec.add_development_dependency "pg"
   spec.add_development_dependency "standard"
   spec.add_development_dependency "webmock"
   spec.add_development_dependency "mysql2"
+
+  spec.add_development_dependency "rspec", "~> 3.10.0"
```

Comment:
Just a heads up, I definitely want to stick with just using the Rails test suite, not Rspec.